bin/webpack-entries.js

```diff
@@ -50,6 +50,9 @@ const blocks = {
 	'product-category': {},
 	'product-categories': {},
 	'product-collection': {},
+	'no-results': {
```

Comment:
May be worth adding a prefix suggesting this block belongs to Product Collection. What do you think?

```suggestion
	'product-collection/no-results': {
```

Reply:
Agreed! Made the changes in commit https://github.com/woocommerce/woocommerce-blocks/pull/11783/commits/9dc01a9b4adce5cde28645de37bc4d791bd8cbcd.